gputop/remote/ajax/metrics.html

```diff
@@ -78,7 +78,12 @@ <h2 id="page-header-metrics">Metrics panel loading</h2>
         var symbol_name = $counter.attr("symbol_name");
         var description = $counter.attr("description") + " (" + $counter.attr("mdapi_usage_flags") + ")";
         var flags = $counter.attr("mdapi_usage_flags").split(" ");
-        set_array[set_array.length - 1].counters.push(new count(name, symbol_name, flags));
+        for (var key in set_array)
+        {
+            if (set_array.hasOwnProperty(key))
+                selected_index = key;
+        }
+        set_array[selected_index].counters.push(new count(name, symbol_name, flags));
```

Comment:
Yeah, it was to just init selected_index to the last key. I think you are correct. Although i'm not 100% sure whether the object is un/ordered. I added a separate array to store the keys inorder.